config/software/supervisor.rb

```diff
@@ -1,5 +1,5 @@
 name "supervisor"
-default_version "3.3.3"
+default_version "4.2.1"
```

Comment:
This seems a big version bump but:
In the agent container, we are already packaging a much more recent version of `supervisor`:
```
$ docker run -ti --rm datadog/agent:7.22.1 supervisord --version
4.1.0
```

Reply:
The version of `supervisor` that is shipped with the agent is in fact coming from:
https://github.com/DataDog/integrations-core/blob/master/supervisord/requirements.in#L2
```
supervisor==4.1.0
```

So, the version in this file doesn’t matter.